pages/linux/pstree.md

```diff
@@ -0,0 +1,11 @@
+# pstree
+
+> A convinient tool to show running processes as a tree.
+
+- Display a tree of processes:
+
+`pstree`
+
+- Display a tree of processes with PIDs:
+
+`pstree -p`
```

Comment:
Also add `pstree {{user_name}}` to show all process trees rooted at processes owned by that user